src/browser/request.ts

```diff
@@ -106,7 +106,7 @@ async function startFetchRequest(
     () => controller?.abort(),
   );
   const headers: { [key: string]: any } = {};
-  for (const headerName of res.headers.keys()) {
+  for (const headerName in res.headers) {
```

Comment:
the previos `for ... of` loop was invalid TS code (`res.headers.keys()`), found after updating the tsconfig file and the typecheck script.

Reply:
nvm, had to revert this change as it broke browser tests.